osmdroid-android/src/main/AndroidManifest.xml

```diff
@@ -8,5 +8,8 @@
 	<uses-feature android:name="android.hardware.telephony" android:required="false" />
 	<uses-feature android:name="android.hardware.wifi" android:required="false" />
 
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+	<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+
```

Comment:
This was required to make lint happy. And it makes sense !

Reply:
No, it doesn't. Those permissions are only needed for some features of osmdroid. If an app doesn't use these, the library should not force the app to require those permissions! To make Lint happy, just ignore the warnings, since it's clear that it's the client apps responsibility to get these permissions if it uses e.g. the mylocationoverlay.

Reply:
with api >= 26 it will be never used, if you don't ask user for permission.
It's an automatism, to reduce maintenance of app which uses this library.

Btw, if you really want to remove it use

```
<uses-permission
        android:name="android.permission.ACCESS_COARSE_LOCATION"
        tools:node="remove" />
```

Reply:
Give me the right to push branches on upstream repo, then you can transform everything on your own flavor !

Reply:
> with api >= 26 it will be never used, if you don't ask user for permission.
> It's an automatism, to reduce maintenance of app which uses this library.

But the app will still inherit these permissions and they will still show up e.g. in playstore, right? Or did something change here, that I'm not aware of?
If not, I still think it's unacceptable to impose these permissions to an app, even if it does not use them.

Reply:
> But the app will still inherit these permissions and they will still show up e.g. in playstore, right? Or did something change here, that I'm not aware of?

Removed means removed

Reply:
Yes, I understand what the second part does.
What I want to say is that the default should not be to just include these permissions. But that an app developer that is using the library AND is using features that require these permissions should be responsible for adding these to his own manifest.
Otherwise everyone who uses the library and does not need location access needs to explicitly remove them or has to answer questions why her app requires a permission that is not obviously needed and has serious privacy implications. I think the default should be the least permissions necessary.
This is also inline with the android sdk: If you want to use LocationManager it's your job as an app developer to make sure you get the necessary permissions....